src/snackbar.jsx

```diff
@@ -78,8 +131,14 @@ const Snackbar = React.createClass({
   },
 
   getInitialState() {
+    let open = this.props.open;
+
+    if (open === null) {
+      open = this.props.openOnMount;
+    }
+
     return {
-      open: this.props.openOnMount,
+      open: open,
```

Comment:
that won't check explicitly for null, it will coerce it to bool. 

maybe:
```js
const {
  open,
  openOnMount,
  message,
  action,
} = this.props;

return {
  action,
  message,
  open: open !== null ? openOnMount : open,
  muiTheme: this.context.muiTheme ? this.context.muiTheme : ThemeManager.getMuiTheme(DefaultRawTheme),
};
```